specification/definitions/Shipment.json

```diff
@@ -26,6 +27,11 @@
       ],
       "additionalProperties": false,
       "properties": {
+        "myparcelcom_shipment_id": {
+          "type": "string",
+          "example": "mp234f21-f350-405f-bf02-1c45d1c792e7",
```

Comment:
I would prefer to have a realistic example here, including the uuid format. If microservice devs store our ID, they improve performance when their database uses an uuid column.
```
      "type": "string",
      "format": "uuid",
      "example": "7b808eee-bf1c-40cd-98f2-3c335a06417e"
```
To avoid confusion, we can change the shipment identifier used by the carrier to something non-uuid like `123456789abcd`